docs/getting_started/installation.md

```diff
@@ -52,32 +52,9 @@ conda activate geograypher
 > chmod ugo+x <CONDA ENV LOCATION>/bin/python3.9
 > ```
 
-If you haven't already, install [poetry](https://python-poetry.org/docs/):
-
-```
-curl -sSL https://install.python-poetry.org | python3 -
-```
-
-Now use this to install the majority of dependencies. First `cd` to the directory containing the `geograypher` repo. Then run:
-
-```
-poetry install
```

Comment:
The whole `poetry` workflow needs to be kept for people who want to develop the project. Otherwise, they will only get the released version.

Reply:
Ok, so for people who want to develop the project they won't have to run `pip install geograypher` right?

Reply:
Correct. That would install a static version which does not correspond to the code they are editing. The only case where a developer could still want to install the static version is to work on examples, knowing that it corresponds to a prior release.